content/docs/user-guide/components/reference/gradients/fastnoise-gradient.md

```diff
@@ -24,15 +44,23 @@ Add the **FastNoise Gradient** component to use one of the noise generation algo
 | Property | Description | Values | Default |
 |-|-|-|-|
 | **Generate Random Seed** | Sets the **Random Seed** property below to a random value. | | |
-| **Preview** | Displays the output gradient of this component after all properties are applied. | | |
+| **Preview** | Displays the output gradient of this component after all properties are applied. Click the previewer icon in the top-left of the preview image to show a larger preview of the gradient in a dockable window. | | |
```

Comment:
> I'm running 22.05 and it's in the top right. (Remember to fix all occurrences like this one.)

I'll change this to "top" so that it will (hopefully) be valid for 22.10 _and_ 22.05.